index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&amp;family=Poppins&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header class="nav">
+        <img class="geom" src="./images/image_geometry_1.svg" alt="geometry">
+        <img class="hamb" src="./images/ic_menu.svg" alt="hamburguer">
+    </header>
+    <main>
+        <section id="greetings">
+            <h1>Juan Pablo Guerrero Rios</h1>
+            <p class="firstp"> Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don't hesitate and contact me.</p>
+            <button class="start">Start colaboration</button>
+            <img class="mix1" src="images/image_geometry_2.svg" alt="mix">
+        </section>
```

Comment:
Please notice that this image ("mix1") shouldn't be an `img` element. Instead, it should be the [background image](https://www.w3schools.com/cssref/pr_background-image.php) of your container. This will make your code semantically accurate and improve your styling abilities 😃. You can [check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more information.
> The same applies to your "geom" image 😉.